### PR TITLE
Humanized string to number 

### LINKED ISSUE #348: Humanized string to number
Is there any chance to convert humanized string to int like this:

``` csharp
var value = "first".ToNumber() // value = 1;
var value = "second".ToNumber() // value = 2;
var value = "17th".ToNumber() // value = 17;
```

?

### PR DESCRIPTION
**Pull request by davidceto1**
fixes #348

### CODE REVIEW

File: src/Humanizer.sln [hunk 1]
```diff
@@ -80,6 +80,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "NuSpecs", "NuSpecs", "{AA44
 		..\NuSpecs\Humanizer.nuspec = ..\NuSpecs\Humanizer.nuspec
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ConsoleApp1", "ConsoleApp1\ConsoleApp1.csproj", "{4F3AF670-463C-4C9D-89B0-F569ECD4D5B9}"
```
Comment (onovotny) [inline src/Humanizer.sln]: Why was this extra project added? Please remove.

File: src/Humanizer.sln [hunk 2]
```diff
@@ -128,6 +130,22 @@ Global
 		{511A7984-F455-4A6E-ADB9-9CAAC47EA079}.Release|x64.Build.0 = Release|Any CPU
 		{511A7984-F455-4A6E-ADB9-9CAAC47EA079}.Release|x86.ActiveCfg = Release|Any CPU
 		{511A7984-F455-4A6E-ADB9-9CAAC47EA079}.Release|x86.Build.0 = Release|Any CPU
+		{4F3AF670-463C-4C9D-89B0-F569ECD4D5B9}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```
Comment (onovotny) [inline src/Humanizer.sln]: These entries should be removed as well.

File: src/Humanizer/Humanizer.csproj [hunk 3]
```diff
@@ -12,5 +12,8 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>Humanizer.snk</AssemblyOriginatorKeyFile>
     <DebugType Condition=" '$(BuildingForLiveUnitTesting)' != 'true' ">embedded</DebugType>
+    <ApplicationIcon />
```
Comment (onovotny) [inline src/Humanizer/Humanizer.csproj]: These three lines should not be here, please remove

**Comment by onovotny** [comment]
As discussed in the original issue, how would this handle a case like `thirtieth` or `seventeenth`?

**Comment by SimonCropp** [comment]
@davidceto1 

given the lack of way to handle "As discussed in the original issue, how would this handle a case like thirtieth or seventeenth?", i will close this one for now. please re-submit if you find a way to handle it

**Pull request closed without merging**